### PR TITLE
Dp sf pap

### PR DESCRIPTION
**Pull request by daniellepace**
Two small fixes (Dec 8th) for dp_sf_pap.

Fixes issue where the new code wouldn't infer medians on the entire 60K dataset. Issue was in the argument parsing.

### CODE REVIEW

File: ml4h/explorations.py [hunk 1]
```diff
@@ -28,26 +29,31 @@
...
 from ml4h.models.legacy_models import legacy_multimodal_multitask_model
+from ml4h.models.model_factory import make_multimodal_multitask_model
 from ml4h.TensorMap import TensorMap, Interpretation, decompress_data
 from ml4h.tensor_generators import TensorGenerator, test_train_valid_tensor_generators
 from ml4h.tensor_generators import BATCH_INPUT_INDEX, BATCH_OUTPUT_INDEX, BATCH_PATHS_INDEX
 from ml4h.plots import plot_histograms_in_pdf, plot_heatmap, plot_cross_reference, SUBPLOT_SIZE
 from ml4h.plots import evaluate_predictions, subplot_rocs, subplot_scatters, plot_categorical_tmap_over_time
 from ml4h.defines import JOIN_CHAR, MRI_SEGMENTED_CHANNEL_MAP, CODING_VALUES_MISSING, CODING_VALUES_LESS_THAN_ONE
 from ml4h.defines import TENSOR_EXT, IMAGE_EXT, ECG_CHAR_2_IDX, ECG_IDX_2_CHAR, PARTNERS_CHAR_2_IDX, PARTNERS_IDX_2_CHAR, PARTNERS_READ_TEXT
+from ml4h.tensorize.tensor_writer_ukbb import _unit_disk
```
Comment (lucidtronix) [inline ml4h/explorations.py:40]: remove the underscore if we're gonna import it

File: ml4h/explorations.py [hunk 2]
```diff
@@ -691,6 +699,206 @@ def infer_with_pixels(args):
...
+    assert(args.batch_size == 1, 'no support here for iterating over larger batches')
+    assert(len(args.tensor_maps_in) == 1, 'no support here for multiple input maps')
+    assert(len(args.tensor_maps_out) == 1, 'no support here for multiple output channels')
+
+    tm_in = args.tensor_maps_in[0]
+    tm_out = args.tensor_maps_out[0]
+    assert(tm_in.shape[-1] == 1, 'no support here for stats on multiple input channels')
+
+    # don't filter datasets for ground truth segmentations if we want to run inference on everything
+    # TODO HELP - this isn't giving me all 56K anymore
```
Comment (lucidtronix) [inline ml4h/explorations.py:801]: I thought you figured this issue out? if so remove comment.

File: ml4h/models/model_factory.py [hunk 3]
```diff
@@ -31,7 +30,9 @@
 
 BLOCK_CLASSES = {
     'conv_encode': ConvEncoderBlock,
+    'merge_conv_encode': ConvEncoderMergeBlock,
```
Comment (lucidtronix) [inline ml4h/models/model_factory.py:33]: Is this block an encoder or merge block -- do you give this to the arg `--encoder_blocks` or `--merge_blocks`? If it is for `encoder_blocks` then can you rename to `unet_conv_encode`?

**Review by lucidtronix — APPROVED** [review]
Couple nitpicks and questions, but once addressed and tests pass feel free to merge! Excellent work @daniellepace !!!

File: ml4h/explorations.py (hunk 1 above)
Comment (daniellepace) [reply ml4h/explorations.py]: Done!

File: ml4h/explorations.py (hunk 2 above)
Comment (daniellepace) [reply ml4h/explorations.py:801]: This was removed on the other branch (dp_kidney)

File: ml4h/models/model_factory.py (hunk 3 above)
Comment (daniellepace) [reply ml4h/models/model_factory.py:33]: It's for `--merge-blocks`.

**Pull request merged**